Combinatorial_map/test/Combinatorial_map/Combinatorial_map_test.cpp

```diff
@@ -193,8 +198,60 @@ bool test_get_new_mark()
   return true;
 }
 
+bool test_face_graph_wrapper()
+{
+  bool res=true;
+
+  typedef CGAL::Surface_mesh<CGAL::Simple_cartesian<double>::Point_3> SMesh;
+  SMesh m;
+  std::ifstream in1("data/head.off");
```

Comment:
Guillaume: you forgot to commit and push that file `data/head.off`. CTest tests fail with:
```
23/24 Test #2705: execution___of__Combinatorial_map_test ..............***Failed    0.01 sec
Error: impossible to open 'data/head.off'
ERROR during test_face_graph_wrapper.
```

Reply:
Will add.